onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -62,6 +62,56 @@
     : rand_engine_(rd()), input_names_(m.GetInputCount()), input_names_str_(m.GetInputCount()), input_length_(m.GetInputCount()) {
   Ort::SessionOptions session_options;
 
+#ifdef _MSC_VER
```

Comment:
Here and other places #ifdef is not needed. since ToUTF8 string has a non windows version.
Also, it is not about the compiler, but the OS